package/features/steps/cli_steps.py

```diff
@@ -102,7 +102,7 @@ def install_kedro(context, version):
 
 @when('I execute the kedro viz command "{command}"')
 def exec_viz_command(context, command):
-    """Execute Kedro viz command"""
+    """Execute Kedro viz command."""
```

Comment:
I tried to reason around why this could be (the weirdly-capitalized) "Kedro viz" for so long last night lol. The "viz" command belonging to Kedro, something else? :D